pmm-tests/pmm-framework.sh

```diff
@@ -2557,6 +2562,20 @@ setup_docker_compose() {
   sudo chmod +x /usr/bin/docker-compose
 }
 
+setup_mongodb_ssl () {
+  echo "Setting up mongodb ssl"
+  export PWD=$(pwd)
+  setup_docker_compose
+  if [ ! -d "pmm-ui-tests" ]; then
+    git clone --branch PMM-8380 https://github.com/percona/pmm-ui-tests
```

Comment:
shouldn't it be main?

Reply:
oh yes, I will change that, I was testing..thanks